test/remote/README.md

```diff
@@ -140,7 +140,7 @@ export ARGOCD_SERVER=127.0.0.1:4443
 Set the admin password to use:
 
 ```shell
-export ARGOCD_E2E_ADMIN_PASSWORD=$(kubectl get secrets argocd-initial-admin-secret -o jsonpath='{.data.password}'|base64 -d)
+export ARGOCD_E2E_ADMIN_PASSWORD=$(kubectl -n argocd-e2e get secrets argocd-initial-admin-secret -o jsonpath='{.data.password}'|base64 -d)
```

Comment:
The e2e tests are expected to be executing in `aregocd-e2e` namespace. We don't need to add the namespace here.

Reply:
Let's add the namespace.